gotpm/cmd/seal.go

```diff
@@ -108,7 +107,16 @@ Thus, algorithm and PCR options are not needed for the unseal command.`,
 		defer srk.Close()
 
 		fmt.Fprintln(debugOutput(), "Unsealing data")
-		secret, err := srk.Unseal(&sealed, nil)
+
```

Comment:
We need to change the description above to note what the (optional) `--pcrs` flag does.

Reply:
`The optional --pcrs flag allows the user to certify the blob that has been sealed
under the same PCR values as current with the given PCR numbers.`